doc/_static/style.css

```diff
@@ -7,8 +7,6 @@
     --pst-font-family-monospace: 'Source Code Pro', var(--pst-font-family-monospace-system);
     /* colors that aren't responsive to light/dark mode */
     --mne-color-discord: #5865F2;
-    --mne-color-primary: #007bff;
-    --mne-color-primary-highlight: #0063cc;
```

Comment:
these blues don't look good against the new theme. If we really want blue as primary, we could consider swapping the theme's `info` (blue) and `primary` (teal) colors, though there's an a11y question about color difference between link and hover, we'd have to make sure `info` is still different enough from hover color to satisfy WCAG